### PR TITLE
Fix for new optimade

### PR DESCRIPTION
**Pull request by Valexr**

### CODE REVIEW

File: src/stores/optimade.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ type StructuresByProviders = Array<[Types.StructuresResponse[], Types.Provider]>
 
 const getStructuresAll = debounce(
 	(providers: string[], filter: string, page: number, limit: number, batch: boolean) => {
-		return optimade.getStructuresAll(providers, filter, page, limit, batch) || [];
+		return optimade.getStructuresAll({ providers, filter, page, limit, batch }) || [];
```
Comment (blokhin) [inline src/stores/optimade.ts:20]: @Valexr only this is the fix for the new client, right? The other edits are fully about something else.

**Comment by blokhin** [comment]
@Valexr also this should be changed: https://github.com/basf/bscience-gui/blob/62d8987ff210a0280e6b93fbf6658d16fcc5772f/package.json#L122

File: src/stores/optimade.ts (hunk 1 above)
Comment (Valexr) [reply src/stores/optimade.ts:20]: Full PR for new client.

**Comment by Valexr** [comment]
> @Valexr also this should be changed:
> 
> https://github.com/basf/bscience-gui/blob/62d8987ff210a0280e6b93fbf6658d16fcc5772f/package.json#L122

Yep, when `optimade-client` version changed.

**Review by blokhin — APPROVED** [review]

**Pull request merged**